atlas_splitter/utils.py

```diff
@@ -9,8 +9,8 @@
 
 HierarchyDict = Dict[str, Any]
 
-MIN_CUSTOM_ID = 1_000_000_000
-MAX_CUSTOM_ID = 4_000_000_000
+MIN_CUSTOM_ID = 20_000
+MAX_CUSTOM_ID = 30_000
```

Comment:
I agree with creating a static mapping. Given that we want to squeeze snuggly the ids into a smaller range, that would be the simplest, most straightforward, and safest approach.

Reply:
The "static mapping" is described [here](https://docs.google.com/presentation/d/1shlysZb4ZCTH88nCWoaDCzgC-5ZtshVaTe0OtUIyfCo/edit#slide=id.g2bfb4e0ed8c_0_17) and that's what is implemented:
https://github.com/BlueBrain/atlas-splitter/blob/67867d7211404413d12516d83ff7e1297687d204/atlas_splitter/utils.py#L71

Reply:
By static we meant to explicitly define a hardcoded dictionary of acronym to id without any fancy algorithmic mapping like hashing.

Reply:
I have no preference here, I just reported what was agreed in a past Cell Atlas meeting together with Jean-Denis.
(I can't think of a way to define such a hardcoded dictionary of acronyms to ID.)

Reply:
I added the collision check here:
https://github.com/BlueBrain/atlas-splitter/commit/45e6e32d09883803758a2bccdd90692d703b9827
I don't think incrementing by one would be reproducible, since then it depends on the order of how regions are added, which could cause problems.  As such, it's a hard error at the moment.